policy-test/tests/e2e_authorization_policy.rs

```diff
@@ -48,6 +50,132 @@ async fn meshtls() {
     .await;
 }
 
+#[tokio::test(flavor = "current_thread")]
+async fn targets_route() {
+    with_temp_ns(|client, ns| async move {
+        // First create all of the policies we'll need so that the nginx pod
+        // starts up with the correct policy (to prevent races).
+        //
+        // The policy requires that all connections are authenticated with MeshTLS.
+        let (srv, all_mtls) = tokio::join!(
+            create(&client, web::server(&ns)),
+            create(&client, all_authenticated(&ns)),
+        );
+        // Create a route which matches the /allowed path.
+        let (root_route, _roux_route) = tokio::join!(
+            create(
+                &client,
+                http_route(
+                    "root",
+                    &ns,
+                    &srv.name_unchecked(),
+                    "/",
+                    NonZeroU16::new(80).unwrap(),
+                ),
+            ),
+            create(
+                &client,
+                http_route(
+                    "roux",
+                    &ns,
+                    &srv.name_unchecked(),
+                    "/roux",
+                    NonZeroU16::new(80).unwrap(),
+                )
+            )
+        );
+        // Create a policy which allows all authenticated clients
+        create(
+            &client,
+            authz_policy(
+                &ns,
+                "root-authz",
+                LocalTargetRef::from_resource(&root_route),
+                Some(NamespacedTargetRef::from_resource(&all_mtls)),
+            ),
+        )
+        .await;
+
+        // Create the nginx pod and wait for it to be ready.
```

Comment:
Same as above, not sure if it applies but thought I'd point it out.

Reply:
```suggestion
        // Create the web pod and wait for it to be ready.
```